src/screens/Home/components/ChooseLocation/components/MapLabel.js

```diff
@@ -1,38 +1,72 @@
 import React from 'react'
-import { Anchor, Box } from 'grommet'
+import { Anchor, Box, Image } from 'grommet'
 import styled from 'styled-components'
 import { func, shape, string } from 'prop-types'
+import { withResponsiveContext } from '@zooniverse/react-components'
 
 export const StyledAnchor = styled(Anchor)`
-  padding: 0.5em;
   white-space: nowrap;
+  position: relative;
+  text-decoration: none;
 
-  :focus, :hover {
-    background: white;
-    text-decoration: underline;
+  &::after {
```

Comment:
this replaces the typical anchor text-decoration/underline?

Reply:
Yes, it's styled so the container's bottom border can extend seamlessly to the preview thumbnail image on hover.